### PR TITLE
Dev/readthedocs pointer

### PR DESCRIPTION
**Pull request by kfieldho**
Now that the ReadtheDocs auto gen is working, point to it from the README

### CODE REVIEW

**Comment by Purg** [comment]
If/When we have more examples, that file will possibly get very large. Shouldn't split it out into multiple files? Possibly nested in a subdirectory to not messy the directory? Not sure if that's a thing that sphinx is happy doing.

**Comment by Purg** [comment]
For example, could we have a directory of python scripts as examples and reference them within a .rst file, resulting in a generated page like its showing now?

**Comment by aashish24** [comment]
> If/When we have more examples, that file will possibly get very large. Shouldn't split it out into multiple files? Possibly nested in a subdirectory to not messy the directory? Not sure if that's a thing that sphinx is happy doing.

I agree with this. Having a single file might not be a good idea as we will have multiple examples.

**Comment by Purg** [comment]
Oh, yeah. Started maintaining release notes files in ``docs/release_nodes/``. Please update the ``since-last-release.md`` file with notes pertaining to features added/updated and bugs fixed.

**Comment by kfieldho** [comment]
Seems like a textbook case of premature optimization.  When we have more examples we should refactor as necessary with the added advantage that we'll know why we're refactoring.

**Comment by kfieldho** [comment]
Re-added the examples directory

File: docs/index.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Contents:
 
    building
    architecture
-   examples
+   docs/examples
```
Comment (Purg) [inline docs/index.rst]: I think you meant to create an ``examples`` directory but instead created a second ``docs`` directory?

**Comment by kfieldho** [comment]
Closing this so that I can deal with fixing the docs as part of this.

**Pull request closed without merging**